ontology/cheminf-core.owl

```diff
@@ -4721,6 +4729,99 @@ The formal charge of any atom in a molecule can be calculated by the following e
+        <obo:IAO_0000118 rdf:datatype="&xsd;string">DBE, degree/level of unsaturation, unsaturation index, index of hydrogen deficiency (IHD)</obo:IAO_0000118>
+        <dc:creator rdf:resource="http://orcid.org/0000-0002-4378-6061"/>
+        <dc:date rdf:datatype="&xsd;dateTime">2022-07-11T08:07:03Z</dc:date>
+        <dc:description rdf:datatype="&xsd;string">The number of molecules of H2 that would have to be added to a molecule to convert all pi bonds to single bonds, and all rings to acyclic structures.</dc:description>
+        <dc:source rdf:datatype="&xsd;anyURI">http://www.chem.ucla.edu/~harding/IGOC/D/double_bond_equivalent.html</dc:source>
+        <rdfs:label xml:lang="en">double bond equivalent</rdfs:label>
+    </owl:Class>
+
+
+
+    <!-- http://semanticscience.org/resource/CHEMINF_000522 -->
+
+    <owl:Class rdf:about="&resource;CHEMINF_000522">
+        <rdfs:subClassOf rdf:resource="&resource;CHEMINF_000031"/>
+        <obo:IAO_0000118 rdf:datatype="&xsd;string">SSSR</obo:IAO_0000118>
+        <dc:creator rdf:resource="http://orcid.org/0000-0002-4378-6061"/>
+        <dc:date rdf:datatype="&xsd;dateTime">2022-07-11T08:18:30Z</dc:date>
+        <dc:description rdf:datatype="&xsd;string">The Smallest Set of Smallest Rings (SSSR) is a subset of cycles within a molecular graph (ring). SSSR was the first widely-used cycle set to classify cyclic molecules in cheminformatics. It offers a limited solution to the problem of molecular cycle perception. Unlike the set of all cycles, an SSSR can filter useless or interfering envelope cycles, but offers no guarantee of uniqueness. A given molecular graph can produce multiple SSSRs with no constraints on which one will be obtained.</dc:description>
+        <dc:source rdf:datatype="&xsd;anyURI">https://depth-first.com/articles/2020/08/31/a-smallest-set-of-smallest-rings/</dc:source>
+        <rdfs:label xml:lang="en">smallest set of smallest rings</rdfs:label>
+    </owl:Class>
+
+
+
+    <!-- http://semanticscience.org/resource/CHEMINF_000523 -->
+
+    <owl:Class rdf:about="&resource;CHEMINF_000523">
+        <rdfs:subClassOf rdf:resource="&resource;CHEMINF_000031"/>
+        <obo:IAO_0000118 rdf:datatype="&xsd;string">index name</obo:IAO_0000118>
+        <dc:creator rdf:resource="http://orcid.org/0000-0002-4378-6061"/>
+        <dc:date rdf:datatype="&xsd;dateTime">2022-07-11T08:20:10Z</dc:date>
+        <dc:description rdf:datatype="&xsd;string">A molecular entity name that represents the name of a molecular structure determined by the naming algorithms developed by ACD/Labs.</dc:description>
+        <dc:source rdf:datatype="&xsd;anyURI">https://www.acdlabs.com/products/name/</dc:source>
+        <rdfs:label xml:lang="en">ACD/Labs index name</rdfs:label>
+    </owl:Class>
+
+
+
+    <!-- http://semanticscience.org/resource/CHEMINF_000524 -->
+
+    <owl:Class rdf:about="&resource;CHEMINF_000524">
+        <rdfs:subClassOf rdf:resource="&resource;CHEMINF_000038"/>
```

Comment:
I don't think there is a separate specification for this. It's part of the same InChI specification.

Do you mean an InChI calculated with with the `FixedH` option? If so, then it should subclass `CHEMINF_000113`.

Reply:
Although my colleague @hujo91 made this and he's currently still on vacation, rereading the definition and its source would lead me to assume that you are right and we're only talking about a differently calculated InChi not the specification. I thus changed it accordingly in [db91e6e](https://github.com/semanticchemistry/semanticchemistry/pull/46/commits/db91e6ef911a34979e5ee17ca9adec8c0b88009c).